### DISCUSSION TITLE
How do I get json information for a package?

### DISCUSSION THREAD

**Discussion by wdlnydhcg** [verdaccio/verdaccio#3154] [Q&A] [answered]
I want to pass a json interface to get the package information, like this: https://registry.npmjs.org/@vue/cli. I deployed a private Verdaccio, but I don't know if such an interface is available

**Comment by juanpicado** [comment, marked as answer]
Same eg: https://registry.verdaccio.org/@vue/cli

**Reply by wdlnydhcg** [reply]
You're right, but it's not what I want. I locally deploy a Verdaccio service https://18.195.155.90/, where I want to get the information package. But I don't know how to get it.

**Reply by juanpicado** [reply]
I'm not sure if I fully understand your question maybe you can elaborate a bit more.

**Reply by wdlnydhcg** [reply]
I built a Verdaccio for private package management, the address is https://18.195.155.90/, now I have an application that needs to get the latest version number of a package, now I don't know how to get it.

**Reply by juanpicado** [reply]
Yeah I got that, but what have you tried so far? or read this ? https://verdaccio.org/docs/cli-registry

**Reply by wdlnydhcg** [reply]
oh，thank you very much. I see how to deal with it

**Reply by joergrech** [reply]
Hi, it looks like the JSON is not accessible when access is set to `$authenticated`. When logged-in I can view private packages and their raw data - but the JSON "page" gives me:
```
{
  "error": "authorization required to access package @test/npm-test-package"
}
```
same problem with "uplinked" packages (e.g., '/@vue/cli').

**Reply by juanpicado** [reply]
When you logged in in the UI, the server gives to the browser a token, this token is stored in the session storage and send it on every request int he UI when you navigage, if you acess directly like `http://xxxxx.verdaccio.domain/jquery` you are not sending any token, so the registry will deny the access.

I am explaining how the token logic works (not for the UI but is almost the same) on my next talk in case you are interested https://geekle.us/schedule/nodejs (it's free to watch btw)
